test/utils.ts

```diff
@@ -0,0 +1,46 @@
+import crypto from "crypto";
+import { mkdirSync, writeFileSync, rmSync } from "fs";
+import { resolve } from "path";
+
+const tempPath = resolve(__dirname, "../node_modules/.tmp/test-mocks");
+
+export function setupWebhooks() {
+  mkdirSync(tempPath, { recursive: true });
+}
+
+export function cleanupWebhooks() {
+  rmSync(tempPath, { recursive: true });
+}
+
+type PullRequestWebhookOptions = {
+  number?: number;
+  state?: "open" | "closed";
+  body?: string;
+  fork?: boolean;
+};
+
+export function generatePullRequestWebhook({
```

Comment:
This generates a JSON file with the event payload to mimic what GitHub does when running a GitHub Action.